### DISCUSSION TITLE
Trimming procesess

### DISCUSSION THREAD

**Discussion by Verinikita** [milaboratory/mixcr#1262] [Q&A] [answered]
Dear MiXCR team

We are running MiXCR software with the best bulk RNAseq preset. Our input data is already trimmed.
We would like to know if it is possible to run the RNAseq-full-length preset analyzer without the intrinsic trimming process.

Best regards,

Veronica

**Comment by Alex-Davydov** [comment, marked as answer]
Hi,
Yes, the option `-Malign.trimmingQualityThreshold=0` can be used to disable trimming during the alignment process.
`mixcr analyze rna-seq -s species -Malign.trimmingQualityThreshold=0 R1.fastq.gz R2.fastq.gz output/`
